src/web/browser/App.tsx

```diff
@@ -58,6 +60,17 @@ const App = ({ CAPI, NAV }: Props) => {
         callFetch();
     }, []);
 
+    useEffect(() => {
+        const callFetch = async () =>
+            setCommentCount(
+                await getCommentCount(
+                    CAPI.config.ajaxUrl,
+                    CAPI.config.shortUrlId,
+                ),
+            );
+        callFetch();
+    }, [CAPI.config.ajaxUrl, CAPI.config.shortUrlId]);
```

Comment:
we want to move away from `CAPI.config` I understand it should be refactored in another PR, so I guess its ok

Reply:
Yeah, I thought that when using it but backed off changing it in favour of doing the work later when relevant